docs/content/index.md

```diff
@@ -0,0 +1,2062 @@
+# API Reference
```

Comment:
are we versioning generated doc?

Reply:
We now generate that file in the workflow